__tests__/allTests.test.js

```diff
@@ -0,0 +1,233 @@
+const app = require('../app')
+const request = require('supertest')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+const endpoints = require('../endpoints.json')
+
+beforeEach(() => seed(testData))
+
+afterAll(() =>db.end())
+
+
+
+describe('/api/topics', () => {
+    test('GET:200 sends an object of topics to the client', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body}) =>{
+            //console.log(body)
+            expect(body.topics).toHaveLength(3)
+            body.topics.forEach((topic) => {
+               // console.log(topic)
+
+                expect(topic).toMatchObject({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                })
+            })
+
+        });
+    })
+})
```

Comment:
Nice test for topics here!

I would remove the commented out console logs once you're done debugging - the version of the code that makes it on to main should be clean